toolkit/offchain/src/reserve/handover.rs

```diff
@@ -79,10 +78,10 @@ pub async fn handover_reserve<
 	Ok(McTxHash(tx_id))
 }
 
-fn get_amount_to_release(reserve_utxo: &ReserveUtxo) -> Option<TokenAmount> {
+fn get_amount_to_release(reserve_utxo: &ReserveUtxo) -> TokenAmount {
 	let token = reserve_utxo.datum.immutable_settings.token.clone();
-	let amount = reserve_utxo.utxo.get_asset_amount(&token).try_into().ok()?;
-	Some(TokenAmount { token, amount })
+	let amount = reserve_utxo.utxo.get_asset_amount(&token);
```

Comment:
Option was an effect of try_into that was from u64 to u64.

Reply:
I think it was also pointed by the clippy

Reply:
```
warning: useless conversion to the same type: `u64`
  --> toolkit/offchain/src/reserve/handover.rs:84:15
   |
84 |     let amount = reserve_utxo.utxo.get_asset_amount(&token).try_into().ok()?;
   |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: consider removing `.try_into()`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#useless_conversion

```